src/editors/playgroundController.ts

```diff
@@ -76,6 +85,34 @@ export default class PlaygroundController {
         log.info('Active editor path', editor.document.uri?.path);
       }
     });
+
+    vscode.window.onDidChangeTextEditorSelection((editor) => {
+      this._selection = editor.selections
+        .map((item, index) => {
+          if (index === 0) {
+            this.showCodeLensForSelection(item);
+          }
+
+          return this.getSelectedText(item);
+        })
+        .join('\n');
+    });
+  }
+
+  public showCodeLensForSelection(item: vscode.Range): void {
+    const selectedText = this.getSelectedText(item).trim();
+    const firstSelectedLine =
+      this._activeTextEditor?.document.lineAt(item.start.line).text.trim() ||
+      '';
+
+    if (
+      selectedText.length > 0 &&
+      selectedText.length >= firstSelectedLine.length
```

Comment:
One small change we could do for usability, is making this case show a code lens:
<img width="721" alt="Screen Shot 2020-07-09 at 1 05 31 PM" src="https://user-images.githubusercontent.com/1791149/87032458-f24c1c80-c1e4-11ea-8558-6edd38e8aaf2.png">
I think if we change this conditional to see if the first selected line selection is empty might cover it.